docs/storage_policy.md

```diff
@@ -44,10 +44,8 @@ Update Existing Assignment
 Updating a storage policy assignment is done by calling `storage_policy.update_info(update_item)`.
 
 ```python
-storage_policy_assignment_id = '1234'
-new__storage_policy_id = '5678'
-new_storage_policy = client.storage_policy('5678')
-updated_storage_policy_assignment = client.storage_policy_assignment(storage_policy_assignment).update_info(new_storage_policy)
+updated_storage_policy = {'storage_policy': {'type': 'storage_policy', 'id': '1234'}}
```

Comment:
We might want to consider creating an override for this method to enable just passing in a StoragePolicy (since that's the only thing that can be updated)